dlaas/setup.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# Some defaults
+ALL_BUNDLES=""
+REQUIREMENTS="requirements.txt"
+SWUPD_ARGS=${SWUPD_ARGS:-""}
+
+# reads a file and sets ALL_BUNDLES array
+get_bundles() {
+    local requirements=$1
+    local IFS=$'\n'
+    [ -e $requirements ] && { ALL_BUNDLES=( $(< $requirements) ); } ||
+	ALL_BUNDLES=()
+}
+
+# install bundles from ALL_BUNDLES array
+install_bundles() {
+    for bundle in ${ALL_BUNDLES[@]}; do
+	if [[ ! $(swupd bundle-list | grep ${bundle}$) ]]; then
+            swupd bundle-add --skip-diskspace-check ${bundle}
+	else
+	    echo "Skipping, bundle '$bundle' already installed"
+	fi
+    done
+}
+
+echo "Updating clear OS..."
+swupd update ${SWUPD_ARGS}
+
+# Get and install bundles: ALL_BUNDLES
+get_bundles $REQUIREMENTS
+if [ ${#ALL_BUNDLES[@]} -ne 0 ]; then
+    echo "Installing clear packages..."
+    install_bundles
+fi
```

Comment:
I don't really see a use for this wrapper script in the repo. I'd prefer to just have the bundles and the install in the Docker file directly. It looks much easier to me than a bunch of extra code as the case of bundle already being installed isn't a worry for doing the image builds.